12/umbraco-cms/reference/configuration/runtimeminificationsettings.md

```diff
@@ -16,10 +16,14 @@ This section allows you to configure the runtime minifications (defaults shown),
   }
 }
 ```
-## Use 'in memory' cache 
+## Use 'in memory' cache
 
 This setting determines whether Smidge should save it's cached output in memory, or in a file on disk. If set to false, then the folder will be created at the wwwroot of your Umbraco site in a folder called 'Smidge'/
 
+{% hint style="info" %}
+It is not possible to disable in memory caching when `Timestamp` is chosen as `CacheBuster` method.
```

Comment:
How about we do something like this instead?
It would possibly also make more sense to have this warning along with the `CacheBuster` method, as that's the value that affects the in-memory cache.

What do you think?

```suggestion
It is not possible to disable in-memory caching when the `CacheBuster` method has `Timestamp` as the value.
```